tools/gen_translations.py

```diff
@@ -210,11 +214,20 @@ def build_grib_cf_map(fuseki, filename, now, git_sha):
 
 
 def main():
+    desc = 'Update translations from metarelate into your source tree'
+    parser = argparse.ArgumentParser(desc)
+    helptxt = ('The location of your local checkout of iris-grib checkout'
+               ' where the updated translations are to be stored.')
+    parser.add_argument('--iris_grib', action='store', help=helptxt)
+    args = parser.parse_args()
     now = datetime.utcnow().strftime('%d %B %Y %H:%m')
     git_sha = requests.get('http://www.metarelate.net/metOcean/latest_sha').text
     with FusekiServer() as fuseki:
-        build_um_cf_map(fuseki, FILE_UM_CF, now, git_sha)
-        build_grib_cf_map(fuseki, FILE_GRIB_CF, now, git_sha)
+        if args.iris_grib:
+            build_grib_cf_map(fuseki, now, git_sha, args.iris_grib)
+        else:
+            build_um_cf_map(fuseki, FILE_UM_CF, now, git_sha)
```

Comment:
Could this not be:
```
build_um_cf_map(fuseki, FILE_UM_CF, now, git_sha)
if args.iris_grib:
    build_grib_cf_map(fuseki, now, git_sha, args.iris_grib)
```
It would save us from having to run this twice (once for the um-cf map and once for the grib-cf map).